openwisp_monitoring/monitoring/base/models.py

```diff
@@ -154,6 +181,9 @@ def write(self, value, time=None, database=None, check=True, extra_values=None):
         """ write timeseries data """
         values = {self.field_name: value}
         if extra_values and isinstance(extra_values, dict):
+            for key in extra_values.keys():
+                if key not in self.related_fields:
+                    raise ValueError(f'{key} not defined in metric configuration')
```

Comment:
I'd add double quotes around {key} to make it easier to understand when reading the error message that key is a configuration value